dftimewolf/lib/collectors/grr_base.py

```diff
@@ -61,13 +61,17 @@ def _check_approval_wrapper(self, grr_object, grr_function, *args, **kwargs):
       The return value of the execution of grr_function(*args, **kwargs).
     """
     approval_sent = False
-    object_id = getattr(grr_object, 'hunt_id') or getattr(grr_object, 'flow_id')
+    object_id = getattr(grr_object, 'hunt_id', None)
```

Comment:
Can you check the type of grr_object here instead? This looks pretty error prone. This would also let you raise if you can't get an object ID.

Reply:
object_id is just used to display an error message. This seems too complex for very little added value, so I'll just remove it.